app/templates/components/large-search.hbs

```diff
@@ -68,5 +68,7 @@
     {{/search-result-set}}
   {{else if search.isRunning}}
     {{dot-spinner}}
+  {{else if (and search.performCount query.length)}}
+    <h4 class="result-info test-result-info">No results for "{{query}}"</h4>
```

Comment:
The `test-` classes are meant to be unique, at least per route. `test-result-info` shouldn't be repeated here.